wandb/sdk/wandb_init.py

```diff
@@ -1154,9 +1155,13 @@ def init(
 
     kwargs = dict(locals())
 
-    # convert fork_from into a version that can be passed to settings
-    if fork_from is not None and resume is not None:
-        raise ValueError("Cannot specify both `fork_from` and `resume`")
+    num_resume_options_set = sum(
+        kwargs.get(k) is not None for k in ["fork_from", "resume", "resume_from"]
+    )
```

Comment:
`kwargs = dict(locals())` is an anti-pattern that we should be avoiding and are trying to remove.

```suggestion
    num_resume_options_set = (
        (fork_from is not None)  # wrap
        + (resume is not None)
        + (resume_from is not None)
    )
```